docs/trace/customizing-the-sdk/README.md

```diff
@@ -359,10 +359,135 @@ using OpenTelemetry;
+`IServiceCollection` and supports a wide range of APIs to enable what is
+generally known as [dependency
+injection](https://learn.microsoft.com/dotnet/core/extensions/dependency-injection).
+
+For the below examples imagine an exporter with this constructor:
+
+```csharp
+public class MyCustomExporter : BaseExporter<Activity>
+{
+    public MyCustomExporter(
+        ILogger<MyCustomExporter> logger,
+        MyCustomService myCustomService)
+    {
+        // Implementation not important
+    }
+}
+```
+
+We want to inject the `ILogger<MyCustomExporter>` and `MyCustomService`
+dependencies into our `MyCustomExporter` instance.
+
+### Using Sdk.CreateTracerProviderBuilder()
+
+To register `MyCustomExporter` and `MyCustomService` we can use the
+`ConfigureServices` and `AddExporter` methods:
+
+```csharp
+using var tracerProvider = Sdk.CreateTracerProviderBuilder()
+    .ConfigureServices(services =>
+    {
+        services.AddLogging();
+        services.AddSingleton<MyCustomService>();
+    })
+    .AddExporter<MyCustomExporter>(ExportProcessorType.Batch)
+    .Build();
+```
+
+When using the `Sdk.CreateTracerProviderBuilder` method the `TracerProvider`
+owns its own `IServiceCollection`. It will only be able to see services
+registered into that collection.
+
+**Note** It is important to correctly manage the lifecycle of the
+`TracerProvider`. See [Building a TracerProvider](#building-a-tracerprovider)
+for details.
+
+### Using the OpenTelemetry.Extensions.Hosting package
+
+**Note** If you are authoring an ASP.NET Core application or using the [.NET
+Generic Host](https://learn.microsoft.com/dotnet/core/extensions/generic-host)
+the
+[OpenTelemetry.Extensions.Hosting](../../../src/OpenTelemetry.Extensions.Hosting/README.md)
+package is the recommended mechanism.
+
+```csharp
+var appBuilder = WebApplication.CreateBuilder(args);
+
+appBuilder.Services.AddSingleton<MyCustomService>();
+
+appBuilder.Services.AddOpenTelemetryTracing(builder => builder
+    .AddExporter<MyCustomExporter>(ExportProcessorType.Batch));
+```
+
+When using the `AddOpenTelemetryTracing` method the `TracerProvider` does not
+own its `IServiceCollection` and instead registers into an existing collection
+(typically the collection used is the one managed by the application host). The
+`TracerProviderBuilder` will be able to access all services registered into that
+collection. For lifecycle management, an [IHostedService
+](https://learn.microsoft.com/dotnet/api/microsoft.extensions.hosting.ihostedservice)
+is used to automatically start the `TracerProvider` when the host starts and the
+host will automatically shutdown and dispose the `TracerProvider` when it is
+shutdown.
+
+**Note** Multiple calls to `AddOpenTelemetryTracing` will configure the same
+`TracerProvider`. Only a single `TraceProvider` may exist in an
+`IServiceCollection` \ `IServiceProvider`.
+
+### Dependency injection `TracerProviderBuilder` extension methods
```

Comment:
```suggestion
### Dependency injection using `TracerProviderBuilder` extension methods
```

Reply:
I don't really want this section to mix in with the "using" sections. I added a new heading to hopefully make it more clear. It is now like this...

```
### Overview
### Examples
#### Using Sdk.CreateTracerProviderBuilder()
#### Using the OpenTelemetry.Extensions.Hosting package
### Dependency injection `TracerProviderBuilder` extension method reference
### Guidance for library authors
```

How does that look?